src/Pages/Recipe/Recipe.css

```diff
@@ -4,28 +4,24 @@
   padding: 10rem 0;
 }
 
-ul {
-  padding-left: 0px;
-}
-li {
+.ingredientsList {
   list-style-type: circle;
   margin: 0 0 10px 10px;
 }
-
-a {
+.recipeLink a {
```

Comment:
Why do you have both  "a" and "recipeLink"?

Reply:
I specify because the summary block That we got from the database is a whole bunch of HTML(p, bold, a …), and I need to specify what part exactly I need to change. If I do not specify, the style applies to the whole block.